src/components/learner-credit-management/members-tab/GroupMembersCsvDownloadTableAction.jsx

```diff
@@ -50,9 +54,13 @@ const GroupMembersCsvDownloadTableAction = ({
       }
     });
 
+    const selectedEmails = [];
+    tableInstance.selectedFlatRows.forEach((row) => selectedEmails.push(row.id));
```

Comment:
[nit/suggestion] then down here, we can just check for the `selectedEmailCount` and add it to the options if individual emails were selected.

`    if (selectedEmailCount > 0) {
      options.learners = selectedEmails;
    }
`